### ISSUE TITLE
User-Task-ID header message on non CORS installs

### ISSUE THREAD

**Issue by bluebob** [linkedin/cruise-control-ui#36]
Setup is CC 0.1.68 on RHEL hosts, with a separate server hosting Nginx and CC-UI (_from the 0.3.0 release tarball_). I've followed the instructions for a non-CORS install (after reading the advice about CORS in issue #7 . This seems to work fine, apart from the "User-Task-ID header..." red banner being printed on the Cluster Load and Control Proposals pages.

I've tried switching to CORS (following the instructions for that setup) and while my CC's look good, I just get the _Network error_ message given in issue #7 *and* that banner in the UI.

If non-CORS is the recommended way, then is there a way to suppress this (unnecessary?) warning? Especially annoying given that the Insights page is able to figure when CORS is Required or not.

For info, the CORS Options check of one of my CC's returns this:
```< HTTP/1.1 200 OK
< Date: Tue, 17 Sep 2019 15:00:43 GMT
< Access-Control-Allow-Origin: http://server-name-removed
< Access-Control-Allow-Methods: OPTIONS,GET,POST
< Access-Control-Allow-Headers: User-Task-ID
< Access-Control-Allow-Credentials: true
< Access-Control-Max-Age: 1728000
< Content-Length: 0
< Server: Jetty(9.4.18.v20190429)
```
which looks to be what's asked for in the documentation.

**Comment by nareshv** [comment]
Will try to add a preference to show/hide this setting.

**Comment by ishita07-danaher** [comment]
I have enabled CORS and getting this message in the front end : User-Task-ID header is not found in the response from the server. If you are using CORS, please add necessary configuration to your Cruise Control as described in this wiki.


Can you please explain what is happening here.

Also, I am not able to see any exceptions in the logs and rest of the application seems to be working fine.

**Comment by nareshv** [comment]
message shows up with the js library doesn't have access to the headers in the response. please ignore the message for now, if things are looking good.

**Comment by nareshv** [comment]
released v0.3.4 for this
